### PR TITLE
feature(documentation): updated readme file

### LINKED ISSUE #1071: Improve the documentation in the documentation readme
Outdated, so needs additions

### PR DESCRIPTION
**Pull request by IagoMota**
# The Pull Request is ready

- [x] fixes  idrinth-api-bench/issues#1071
- [ ] all actions are passing
- [ ] only fixes a single issue

## Overview

Made it more noob friendly.

## Documentation-Website

- [ ] mobile view is usable
- [ ] desktop view is usable
- [ ] no a-tags are used directly (NavLink, MailLink, ExternalLink instead)
- [ ] all new texts are added to the translation files (at least the english one)
- [ ] tests have been added (if required)
- [ ] shared code has been extracted in a different file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,50 @@ Website made using React to show the ideas behind the API benchmark implementati
...
+
+### Installation command
+
 ```bash
-npm install
+npm ci
 ```
 
 ## Running the app
+If you encounter any problems with this, try checking the `Token tutorial` right above this section.
```
Comment (itsmacr8) [inline README.md]: I think we can make it a link for easier access instead of instructing to scroll. Syntax `[Token tutorial](link)`

**Review by itsmacr8 — CHANGES_REQUESTED** [review]
Consider making the change.

**Review by Idrinth — CHANGES_REQUESTED** [review]
Please add some line breaks at <= 80 characters to your texts, this would fail the markdown standard lint test if that was active

File: README.md (hunk 1 above)
Comment (IagoMota) [reply README.md]: Agreed!

**Comment by IagoMota** [comment]
> Please add some line breaks at <= 80 characters to your texts, this would fail the markdown standard lint test if that was active

Done

**Review by IagoMota — COMMENTED** [review]
@Idrinth should be okay now

**Pull request merged**